src/sentry/mediators/sentry_app_installation_tokens/destroyer.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
+
+from sentry.utils.cache import memoize
+from sentry.mediators import Mediator, Param
+from sentry.models import (
+    AuditLogEntryEvent, SentryAppInstallationToken
+)
+
+
+class Destroyer(Mediator):
+    api_token = Param('sentry.models.ApiToken')
+    generate_audit = Param(bool, default=False)
```

Comment:
When would we _not_ want to generate the audit log?

Reply:
when a token is getting deleted because the install is getting deleted. Since we already have audit logging for when that happens